src/components/Label/_Label.scss

```diff
@@ -7,7 +7,7 @@
     justify-content: space-between;
 
     &__required-helper {
-        font-size: var(--type-size--1);
+        font-size: var(--font-size--1);
```

Comment:
This seemed to be just a variable name that wasn't updated if/when `type` was changed to `font`.